src/Runner.jl

```diff
@@ -1125,6 +1125,12 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
             merge!(mapping, Dict(
                 "RUSTUP_TOOLCHAIN" => "$(rust_version)-$(map_rust_target(host_platform))",
             ))
+            if rust_version ≥ v"1.68.0"
+                merge!(mapping, Dict(
+                    # Use sparse crates.io registry for quicker updates
+                    "REGISTRIES_CRATES_IO_PROTOCOL" => "sparse",
```

Comment:
This doesn't seem to be working as expected.... :sob: :sob: :sob:

Reply:
It turns out the name of the environment variable was misspelled in https://github.com/rust-lang/cargo/pull/11224#issue-1406777084 (see edit history of the message...)